build.gradle

```diff
@@ -187,6 +188,12 @@ dependencies {
     implementation 'com.exactpro.th2:grpc-service-generator:3.2.2'
     implementation "com.exactpro.th2:cradle-cassandra:${cradleVersion}"
 
+    def autoValueVersion = '1.10.1'
+    implementation "com.google.auto.value:auto-value-annotations:$autoValueVersion"
+    kapt "com.google.auto.value:auto-value:$autoValueVersion"
+    //this is required to add generated bridge classes for kotlin default constructors
+    implementation(files("$buildDir/tmp/kapt3/classes/main"))
```

Comment:
Unfortunately, I didn't find a workaround for that. The kapt generates the bridge classes to call the kotlin constructor with default arguments.
This repo does a better job https://github.com/F43nd1r/autodsl but we couldn't use it because it brings new vulnerabilities

Reply:
Actually, this one does it as well...
https://github.com/th2-net/th2-common-j/security/code-scanning?query=is%3Aopen+branch%3Abuilders++

I will try the autodsl and see if the reporting become red

Reply:
It uses reflection under the hood. So, not sure if it is better than the current solution